model/patch_lifecycle.go

```diff
@@ -491,10 +492,27 @@ func FinalizePatch(ctx context.Context, p *patch.Patch, requester string, github
 	}
 	intermediateProject.CreateTime = patchVersion.CreateTime
 
-	manifest, err := constructManifest(patchVersion, project, projectRef, settings)
+	autoUpdateModules := project.GetAutoUpdateModules()
+
+	mfst, err := constructManifest(patchVersion, projectRef, autoUpdateModules, settings)
 	if err != nil {
 		return nil, errors.Wrap(err, "constructing manifest")
 	}
+	if mfst != nil {
```

Comment:
Should we also check that len(autoUpdateModules) != len(project.Modules) ?

Reply:
I don't think so because we should be able to specify a subset of the modules to auto-update